datadistillrSDK/pythonSDK.py

```
@@ -0,0 +1,139 @@
+        login_resp_json = login_response.json()
+        return login_resp_json
+
+    def logout(self):
+        """
+        Log user out of DataDistillr account.
+
+        Returns:
+            json: A json containing account details and login status.
+        """
+
+        logout_response = self.session.get(url=self.logout_page, verify=False)
+        logout_resp_json = logout_response.json()
+        self.is_logged_in = logout_resp_json["loggedIn"]
+        return logout_resp_json
+
+    def get_projects(self):
+        """
+        Gets all projects in DataDistillr account.
+
+        Returns:
+            list<project>: A list of project objects
+        """
+
+        if not self.is_logged_in:
+            return "login is incorrect"
+
+        # URL to get project needs active org token, URL format may change
+        org_token = self.login_resp_json["activeOrganization"]["token"]
+        projects_page = self.base_url + "organization/" + str(org_token) + "/projects"
+        projects_response = self.session.get(url=projects_page, verify=False)
+
+        # Converts response to JSON
+        proj_resp_json = projects_response.json()
+
+        # Gets the projects list
+        proj_list = proj_resp_json["projects"]
+        proj_object_list = []
+
+        for i in range(len(proj_list)):
+            # create new project object with json
+            proj_object = project(proj_list[i], self.session)
+            proj_object_list.append(proj_object)
+
+        return proj_object_list
+
+    def get_project(self, project_name):
+        """
+        Gets an individual project object.
+
+        Parameters:
+            project_name (string): The name of a project in DataDistillr account.
+
+        Returns:
+            project: A project object. If project_name does not match existing projects, it returns "project not found".
+        """
+
+        if not self.is_logged_in:
```

If the user is not logged in, we should throw an exception rather than returning a string.  In general, the function should always return the same thing.  So if the function is expected to return a `project` object, it should always do so or throw an exception, or `null` if there is an error.